src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -1041,4 +1041,41 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
     <value>Warning As Error: {0}</value>
     <comment>{0} is a warning message</comment>
   </data>
+  <data name="Vulnerability_Severity_1" xml:space="preserve">
+    <value>low</value>
+    <comment>As in "This package has a low severity vulnerability"</comment>
+  </data>
+  <data name="Vulnerability_Severity_2" xml:space="preserve">
+    <value>moderate</value>
+    <comment>As in "This package has a moderate severity vulnerability"</comment>
+  </data>
+  <data name="Vulnerability_Severity_3" xml:space="preserve">
+    <value>high</value>
+    <comment>As in "This package has a high severity vulnerability"</comment>
+  </data>
+  <data name="Vulnerability_Severity_4" xml:space="preserve">
+    <value>critical</value>
+    <comment>As in "This package has a critical severity vulnerability"</comment>
+  </data>
+  <data name="Vulnerability_Severity_unknown" xml:space="preserve">
+    <value>unknown</value>
+    <comment>As in "This package has a unknown severity vulnerability"</comment>
+  </data>
+  <data name="Warning_PackageWithKnownVulnerability" xml:space="preserve">
+    <value>Package '{0}' {1} has a known {2} severity vulnerability, {3}.</value>
+    <comment>{0} - package id
+{1} - package version
+{2} - severity, for example low, moder, high, or critical
```

Comment:
```suggestion
{2} - severity, for example low, moderate, high, or critical
```
also should the severities be translated here? Or not, as Error_InvalidNuGetAuditLevelValue is not?

Reply:
I'm not sure I understand the question. The severity here is in the resource comment. The severity is translated in the resources above (Vulnerability_Severity_*), and is substitued for `{2}` as per this resource's comment.

Reply:
You have introduced several strings. In some strings, you ask the translation team NOT to translate the severities. In this one, you do not make that request. So if you intend some strings to translate "moderate" and other strings not to translate "moderate", then I think you've accomplished that.